### PR TITLE
add Chinese readme

### PR DESCRIPTION
**Pull request by cangerjun**
add Chinese readme

### CODE REVIEW

**Review by Specy — COMMENTED** [review]
Leave the README.md in the root of the repository

**Comment by Specy** [comment]
@cangerjun i don't understand, is this PR ready to be merged or is it still a draft?

**Comment by Specy** [comment]
And also, i saw you deleted the old PR that changed the last things that needed to be translated, was it intentional?

File: README-ZH.md
Comment (cangerjun) [inline README-ZH.md:1]: please seave this one flie
this one flie is over.

**Review by cangerjun — COMMENTED** [review]
Now I have cancelled the readme modification in the root directory, and added a light readme zh in the root directory

**Pull request merged**